crates/wysiwyg/src/dom/parser/parse.rs

```diff
@@ -269,19 +269,39 @@ mod sys {
         where
             S: UnicodeString,
         {
-            // TODO add some logic here to determine if it's a mention or a link
-            let attributes = child
-                .attrs
-                .iter()
-                .filter(|(k, _)| k != &String::from("href"))
-                .map(|(k, v)| (k.as_str().into(), v.as_str().into()))
-                .collect();
-
-            DomNode::Container(ContainerNode::new_link(
-                child.get_attr("href").unwrap_or("").into(),
-                Vec::new(),
-                attributes,
-            ))
+
+            // initial implementation, firstly check if we have either `contenteditable=false` or `data-mention-type=`
+            // attributes, if so then we're going to add a mention instead of a link
```

Comment:
I went with this initially to avoid multiple test case failures. I actually think it will be better to use just the presence of `data-mention-type` to determine if a link is a mention or a plain link. I do this further down, just wanted to make sure all the tests were passing for the time being